test_runner/src/main/kotlin/ftl/cli/firebase/test/android/AndroidRunCommand.kt

```diff
@@ -243,7 +243,7 @@ class AndroidRunCommand : CommonRunCommand(), Runnable {
     // AndroidFlankYml
 
     @Option(
-        names = ["--additional-app-test-apks"],
+        names = ["--app-test-apks", "--additional-app-test-apks"],
```

Comment:
we should probably keep the flags 1:1 mapping with the yaml. `--app-test-apks` isn't a valid key name in the yaml.